Dockerfile

```diff
@@ -38,19 +39,20 @@ RUN echo 'Installing extensions' && \
     # contact
     pip install -e git+https://github.com/okfn/ckanext-contact.git@lacounts-0.1#egg=ckanext-contact
 
-
-COPY . /srv/app/src/ckanext-lacounts
+FROM ubuntu:20.04
```

Comment:
I'm unclear why you've added this FROM line; the default product of this Dockerfile is going to be only what's after the last `FROM`

Reply:
you're right. I just learned about that. Fixing it